server/test/test_api.py

```diff
@@ -147,6 +147,34 @@ def test_filter_put_annotations_obs(self):
         self.assertEqual(len(result_data["data"][0]), 3)
         self.assertEqual(len(result_data["data"]), 15)
 
+    def test_diff_exp(self):
+        endpoint = "diffexp/obs"
+        url = f"{URL_BASE}{endpoint}"
+        params = {
```

Comment:
This test does not conform to the spec.    mode should be an object and contain "count" and "name", eg,

{  mode:  { count: 7, name: "topN" } }

Reply:
spec is wrong (inconsistent) - will fix it.